atomic_reactor/plugins/pre_check_and_set_rebuild.py

```diff
@@ -94,15 +93,6 @@ def run(self):
                                       namespace=metadata.get('namespace', None))
             osbs = OSBS(osbs_conf, osbs_conf)
             labels = {self.label_key: self.label_value}
-            try:
```

Comment:
I was thinking the same, but Tim pointed out, that it actually does, search for 'retry_on_conflict'